packages/ckeditor5-dev-env/tests/fixtures/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Don't assume that there will be just on fixture project needed. Wrap it in `basic/` or similar directory.